testfile.txt

```diff
@@ -0,0 +1 @@
+////
```

Comment:
Please align this with the team’s code style guide. For instance, use 2-space indentation here